problem-based-packs/insecure-transport/java-stdlib/httpurlconnection-http-request.java

```diff
@@ -10,8 +10,8 @@ private static void sendbad1() throws IOException {
     }
 
     private static void sendbad2() throws IOException {
-            // ruleid: httpurlconnection-http-request
             String url = "http://example.com";
+            // ruleid: httpurlconnection-http-request
```

Comment:
Note that when we change finding locations this will produce new findings in the App because the [`syntactic context`](https://semgrep.dev/docs/managing-findings/#semgrep-ci) changes. Not a huge deal, but it will produce some cruft for the App and new finding notifications.